services/user-svc/internal/organization/organization.go

```diff
@@ -176,6 +177,50 @@ func (s ServiceServer) GetOrganization(ctx context.Context, req *pb.GetOrganizat
 	}, nil
 }
 
+func (s ServiceServer) GetOrganizationsByUser(ctx context.Context, req *pb.GetOrganizationsByUserRequest) (*pb.GetOrganizationsByUserResponse, error) {
+	db := hwgorm.GetDB(ctx)
+
+	// TODO: Auth
+
+	userID, err := uuid.Parse(req.UserId)
+	if err != nil {
+		return nil, status.Error(codes.InvalidArgument, err.Error())
+	}
+
+	var organizations []Organization
+	err = db.Preload("Organization").Where("user_id = ? OR created_by_user_id = ?", userID, userID).Find(&organizations).Error
```

Comment:
`ERROR: column \"user_id\" does not exist (SQLSTATE 42703)`
You need to query the organizations via the memberships only.